src/utils/AuthorizationPersistance.js

```diff
@@ -0,0 +1,89 @@
+const db = require('../storage/DataAccess');
+
+const SERVERS = 'serverConfig';
+const CLIENTS = 'clientConfig';
+const ACCESS = 'accessTokens';
+const KEYS = 'serverKeys';
+
+/**
+ * Required Fields for AuthorizationPersistance Data Types:
+ *
+ * Server:
+ *  @param {string} id - the id of the server
+ *
+ * Config:
+ *  @param {string} jwks_url - the url for the jwks
+ *
+ * Client:
+ *  @param {string} id - the id of the client
+ *  @param {string} serverId - the id of server (added when client config is saved)
+ *
+ * Token:
+ *  @param {string} jwt - the jwt
+ *  @param {number} exp - the time the token expires (UTC)
+ *  @param {string} serverId - the id of server (added when token is saved)
+ *
+ *  Key:
+ *  @param {string} serverId - the id of server (added when key is saved)
+ *
+ */
+
+class AuthorizationPersistance {
+  addServer(server) {
+    db.insert(SERVERS, server);
+  }
+
+  addServerConfiguration(server, config) {
+    db.upsert(SERVERS, config, s => s.id === server.id);
```

Comment:
This will update the value in the database to just be `config`. But I think in reality it should update to be 
```
{
   ...server
   ...config
}
```